src/main/java/use_cases/login_leaderboard/IFileOutput.java

```diff
@@ -1,7 +1,12 @@
 package use_cases.login_leaderboard;
 
+import java.io.IOException;
+
 /**
- * Update the CSV file with a new user or
+ * Update the CSV file with a new user or score.
  */
 public interface IFileOutput {
```

Comment:
Is this the output boundary? If so, it might be better to change the name to IFileOutputBoundary just to make it obvious and consistent with other classes.